source/core/Timestamp.cpp

```diff
@@ -141,7 +139,7 @@ namespace Vireo
         time_t t = _mkgmtime(&timeVal);
     #endif
     #endif
-        *this = Timestamp((Double)t + kStdDT1970re1904, fracSecs * 18446744073709551616.0);
+        *this = Timestamp((Double)t + kStdDT1970re1904 + fracSecs);
```

Comment:
Please elaborate why this change is needed. This appears to be a change of behavior rather than a warning fix.

Reply:
Yes, this was something I found while going through this file. It is the same fix as my previous timestamp precision fix, where we were losing precision by using the Timestamp constructor with two parameters. I am using the one with a Double parameter here instead, which preserves the precision. I don't know if this manifested as a bug or not.